invictusd0a/ah-frontend
# #123456789 Create empty react repository

**What does this PR do?**
- Create react app using tool create react app
- Customize info and folders. Delete react logo
- Set up for react a test environment

FIle structure
<img width="297" alt="Screen Shot 2019-05-23 at 4 46 16 PM" src="https://user-images.githubusercontent.com/44635784/58257843-4e05dd00-7d7a-11e9-8687-56ee99f487aa.png">

How to test this out?
Clone the repo
Change directory into the react app
Run the following command

`npm start`

In the browser, localhost 3000 should run the server and you should view 

<img width="1222" alt="Screen Shot 2019-05-23 at 4 48 30 PM" src="https://user-images.githubusercontent.com/44635784/58258032-a806a280-7d7a-11e9-9754-f8bdf5bf930a.png">

To run tests, input the command

`npm test`

<img width="319" alt="Screen Shot 2019-05-23 at 5 06 16 PM" src="https://user-images.githubusercontent.com/44635784/58259406-1f3d3600-7d7d-11e9-9dda-7bbfb3f2b667.png">



[Starts #123456789]
